src/cladetime/sequence.py

```diff
@@ -258,7 +260,12 @@ def filter_metadata(
 
 
 def get_clade_counts(filtered_metadata: pl.LazyFrame) -> pl.LazyFrame:
```

Comment:
Left this here because the variant-nowcast-hub scripts still reference it. It's replacement (`summarize clades`) does the same thing but:
-  has a better name
- allows a configurable list of group_by columns